frontend/pages/queries/details/QueryDetailsPage/_styles.scss

```diff
@@ -39,6 +39,10 @@
   &__log-destination {
     display: flex;
     gap: $pad-small;
+
+    .component__tooltip-wrapper__element {
+      font-weight: $bold;
+    }
```

Comment:
can you target this:
<img width="713" alt="Screenshot 2023-10-09 at 1 16 34 PM" src="https://github.com/fleetdm/fleet/assets/61553566/9d084cc5-1b40-4813-9bc6-cdcc289727c4">

Reply:
Done! Sorry!!--thought you meant the Automation tooltip word not within the tooltip!